components/02-molecules/alert/_yds-alert.scss

```diff
@@ -53,6 +54,11 @@ $alert-themes: map.deep-get(sass-tokens.$tokens, color, alert);
     [data-alert-state='expanded'] & {
       align-self: flex-start;
     }
+
+    // fix icon alignment shift
+    [data-alert-state='dismissed'] & {
+      align-self: flex-start;
+    }
```

Comment:
I was curious what these were for, but I played with it, and I see now. I would probably just lump this in with the chunk above, instead of having the two separate declarations. Because really, we want the "expanded" and the "dismissed" states to be the same size - always, and forever - don't ever let them be different. The way you have it now, someone could tweak one size, and not realize they need to change the other. So we can "couple" them by writing it like this instead:

```
[data-alert-state='expanded'] &,
    [data-alert-state='dismissed'] & {
      align-self: flex-start;
    }
```

And then do the same for the icon size.